File: generators/app/templates/src/main/java/package/_Application.java
package <%= groupCases.splitByDot %>.<%= nameCases.splitByDot %>;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
import org.springframework.cloud.netflix.feign.EnableFeignClients;

@EnableEurekaClient
@SpringBootApplication
@EnableFeignClients
public class <%= nameCases.hump %>Application {

    public static void main(String[] args) {
        SpringApplication.run(<%= nameCases.hump %>Application.class, args);
    }
}

File: generators/service/templates/src/main/kotlin/package/_Application.kt
package <%= groupCases.splitByDot %>.<%= nameCases.splitByDot %>

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.netflix.eureka.EnableEurekaClient
import org.springframework.cloud.openfeign.EnableFeignClients

@EnableFeignClients
@EnableEurekaClient
@SpringBootApplication
class <%= nameCases.hump %>Application

fun main(args: Array<String>) {
    runApplication<<%= nameCases.hump %>Application>(*args)
}
